lib/handlers/quote/quote.ts

```diff
@@ -251,6 +257,10 @@ export class QuoteHandler extends APIGLambdaHandler<
 
     const requestSourceHeader = params.event.headers && params.event.headers['x-request-source']
     const appVersion = params.event.headers && params.event.headers['x-app-version']
+    const universalRouterVersion = convertStringRouterVersionToEnum(
+      params.event.headers && params.event.headers['x-universal-router-version']
```

Comment:
nit: something like this maybe `params.event.headers?.['x-universal-router-version']`